psalm.xml

```diff
@@ -15,11 +15,17 @@
         <ignoreFiles>
             <directory name="test/TestAsset"/>
             <directory name="vendor"/>
-            <!-- can be removed once PHP8.0 support dropped -->
-            <directory name="test/Strategy/TestAsset"/>
         </ignoreFiles>
     </projectFiles>
 
+    <issueHandlers>
+        <InternalClass>
+            <errorLevel type="suppress">
+                <file name="test/HydratorPluginManagerFactoryTest.php" />
+            </errorLevel>
+        </InternalClass>
+    </issueHandlers>
+
```

Comment:
FYI, you can use something like this on class level doc blocks to stop Psalm complaining:

```php
/**
 * @psalm-internal \Laminas
 * @psalm-internal \LaminasTest
 */
```